diesel/src/pg/types/money.rs

```diff
@@ -0,0 +1,32 @@
+//! Support for Money values under PostgreSQL.
+use std::error::Error;
+use std::io::prelude::*;
+
+use byteorder::{ReadBytesExt, WriteBytesExt, NetworkEndian};
+
+/// Money is reprsented in Postgres as a 64 bit signed integer. The fractional precision of the
+/// value is determined by the [lc_monetary setting of the database](https://www.postgresql.org/docs/9.1/static/datatype-money.html).
```

Comment:
Also, any reason you link to the 9.1 docs instead of more recent ones? Was this added in 9.1?